requirements/requirements.txt

```diff
@@ -0,0 +1,30 @@
+attrs==21.4.0
```

Comment:
This seems like a lot of packages to be adding - what are they are being used for?

Reply:
did a cleanup